backend/headless/output.c

```diff
@@ -93,8 +93,10 @@ static int signal_frame(void *data) {
 	return 0;
 }
 
+static int current_output_id = 0;
```

Comment:
You can use `wl_list_length` instead of a global.

Reply:
I'm trying to avoid duplicate names because it might be a hard case to handle for some uses. For instance, if I add two backends, delete the first one, then add another one, I have two HEADLESS-1 outputs.